hack/prebuild/main.tf

```diff
@@ -10,6 +10,7 @@ resource "packet_device" "libvirt" {
   operating_system = "centos_7"
   billing_cycle    = "hourly"
   project_id       = "${var.packet_project_id}"
+  tags             = "${list("${var.tag}")}"
```

Comment:
Why is the `${list...}` now required?

Reply:
because `tags` needs a list type, not a string